pandas/tests/series/test_combine_concat.py

```diff
@@ -15,29 +15,28 @@
 from pandas.util.testing import assert_series_equal
 import pandas.util.testing as tm
 
-from .common import TestData
 
+class TestSeriesCombine():
```

Comment:
Only comment I have (applicable to your other PRs) is that I thought that these would have to inherit from `object` for compatibility but looks like tests are passing and this inheritance idiom is used in a few other instances, so may have just been a misunderstanding on my end. If anyone else knows if that's a requirement for something else LMK